src/cmd/tools.go

```diff
@@ -0,0 +1,131 @@
+package cmd
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/opslevel/cli/common"
+	"github.com/opslevel/opslevel-go/v2024"
+	"github.com/spf13/cobra"
+)
+
+var createServiceToolCmd = &cobra.Command{
+	Use:   "tool",
+	Short: "Create service tool",
+	Example: `
+cat << EOF | opslevel create service tool -f -
+service: my-service-alias
+category: deployment
+displayName: "fancy tool"
+environment: "dev"
+url: "https://example.com"
+EOF
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getClientGQL()
+		serviceIdentifier := readServiceFieldFromYaml()
```

Comment:
what about putting it as an `arg` to the command.  I think we do this in other places too - so it would be like this
```
cat << EOF | opslevel create service tool my-service-alias -f -
category: deployment
displayName: "fancy tool"
environment: "dev"
url: "https://example.com"
EOF
```

This would eliminate all that extra reading from yaml stuff

Reply:
Good call out. Adding this in the next commit